interface/wx/textctrl.h

```diff
@@ -219,6 +219,17 @@ enum wxTextAttrLineSpacing
 };
 
 
+/**
+    Those values can be used with SetFontUnderlinedWithEffects
```

Comment:
```suggestion
    The values that can be used with SetFontUnderlinedWithEffects
```
Also don't forget `@since 3.1.3` (here and everywhere below).